ports/opengl/portfile.cmake

```
@@ -26,20 +26,32 @@ if(NOT VCPKG_CMAKE_SYSTEM_NAME OR VCPKG_CMAKE_SYSTEM_NAME STREQUAL "WindowsStore
 
     file(MAKE_DIRECTORY
         ${CURRENT_PACKAGES_DIR}/include/gl
-        ${CURRENT_PACKAGES_DIR}/lib
-        ${CURRENT_PACKAGES_DIR}/debug/lib
         ${CURRENT_PACKAGES_DIR}/share/opengl
     )
+    if (NOT VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL release)
```

```suggestion
    if (NOT VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "release")
```

gotta make sure not to dereference the `release` variable :)